upgrade/lab5.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+echo "ACTIVATING SCENARIO #5...
+"
+
+path="$1"
+sudo $path/upload
+sudo $path/gravity upgrade --manual
+sudo $path/gravity plan execute --phase=/init >/dev/null 2>&1
+sudo $path/gravity plan execute --phase=/checks >/dev/null 2>&1
+sudo $path/gravity plan execute --phase=/pre-update >/dev/null 2>&1
+sudo $path/gravity plan execute --phase=/bootstrap >/dev/null 2>&1
+sudo $path/gravity status-reset --confirm >/dev/null 2>&1
+
+echo "
+SCENARIO #5 ACTIVATED.
+
+OBJECTIVE
+---------
+
+The upgrade process and requirements have now been tampered.
+
+A manual upgrade has been already started, you can check the plan with
+
+sudo ${path}/gravity plan
+
+and
+
+sudo ${path}/gravity status
+
+Your goal is to inspect the upgrade plan and status and fix current issues.
+"
```

Comment:
After upgrade successfully launches should we rollback similar to other scenarios prior to proceeding to next scenario to get to original state? 

`sudo ./gravity rollback`

Reply:
I guess both ways work... depending on how we want to proceed either reaching the end of installation and then uninstalling or rolling back should work, really.

Let's pick a choice and stick with that.

Reply:
Yeah I think rollback might be less time-consuming than full reinstall so I'd prob. suggest that.

Reply:
Yeah for the time constraint I was thinking the same thing, rollback may be more efficient

Reply:
I don't think I've seen that mentioned in other scripts output though.. should we add it only here in 5 or in all the others?

Reply:
ah wait, I saw it in 4 now.. using the same phrasing